pytest.ini

```diff
@@ -39,4 +39,4 @@ flake8-ignore =
     scrapy/utils/markup.py F403
     scrapy/utils/multipart.py F403
     scrapy/utils/url.py F403 F405
-    tests/test_loader.py E741
+    tests/test_loader.py E741
```

Comment:
Please, remove the change here (you removed the empty line at the end of the file, we keep an empty line at the end of each file by convention).

Reply:
Sorry, must have removed it when resolving the merge conflict.